lib/index.ts

```diff
@@ -19,52 +19,65 @@ var XML2JS_OPTS = {
  * @param {object} opt Is the option with the filePath or xmlContent and the optional format.
  * @return {object} The pom object along with the timers.
  */
-function parse(opt, callback) {
+function parse(
+  opt: (Options & { filePath?: string; xmlContent?: string }) | null,
+  callback: (
+    e: Error | null,
+    r?: {
```

Comment:
Let's extract them as `interface` in the file so that we have named types being exported.

Reply:
Makes sense. Made this change. Extracted them as types instead of interfaces, let me know if that's an issue.

Reply:
`type`s are ok.